README.md

```diff
@@ -1,67 +1,79 @@
-# cf-cdn-service-broker
+# Cloud Foundry CDN Service Broker
 
-A Cloud Foundry service broker for CloudFront and Let's Encrypt
+A [Cloud Foundry](https://www.cloudfoundry.org/) [service broker](http://docs.cloudfoundry.org/services/) for [CloudFront](https://aws.amazon.com/cloudfront/) and [Let's Encrypt](https://letsencrypt.org/).
 
 ## Deployment
 
-* Create services
+1. Clone this repository, and `cd` into it
+1. Target the space you want to deploy the broker to
 
-    Note: the following credentials must be available in the `cdn-creds` service or in top-level environment variables:
+    ```bash
+    $ cf target -o <org> -s <space>
+    ```
 
-    * PORT
-    * BROKER_USER
-    * BROKER_PASS
-    * DATABASE_URL
-    * EMAIL
-    * ACME_URL
-    * BUCKET
-    * AWS_ACCESS_KEY_ID
-    * AWS_SECRET_ACCESS_KEY
-    * AWS_DEFAULT_REGION
+1. Create a [user-provided service](http://docs.cloudfoundry.org/devguide/services/user-provided.html) with the configuration (you will be prompted for the values of each)
```

Comment:
I like user-provided too but the idea of a laptop deploy is not a good one :p 

The good thing about concourse having the secrets is that it can deploy the apps with or without a previous instantiation of them. I'd say lets keep it here for now...